kernelci.org/content/en/docs/instances/staging.md

```diff
@@ -80,7 +80,7 @@ There is a timer on the staging.kernelci.org server which starts a job every
 1. update the `kernelci-backend` service using Ansible from [`kernelci-backend-config`](https://github.com/kernelci/kernelci-backend-config) with the staging branch
 1. update [staging branch for `kernelci-frontend`](https://github.com/kernelci/kernelci-frontend/tree/staging.kernelci.org)
 1. update the `kernelci-frontend` service using Ansible from [`kernelci-frontend-config`](https://github.com/kernelci/kernelci-frontend-config) with the staging branch
-1. create and push a `staging.kernelci.org` branch with a tag to the [KernelCI
+1. create and push a `staging-mainline` branch with a tag to the [KernelCI
```

Comment:
Yes, this one is correct. However, it rotates every 8h between `staging-mainline`, `staging-stable` and `staging-next` so we get the 3 flavours covered every day. Maybe that should also be mentioned in the documentation, even though it's a bit more complicated to explain.

Reply:
Ok, thanks.  Is the schedule for the job that runs every 8h committed to source control somewhere?  (I'd like to understand that before considering other ways to suggest changes to this documentation)

Reply:
This script is run every 8h:
https://github.com/kernelci/kernelci-deploy/blob/main/staging.kernelci.org

There's a systemd timer on the staging server to run it, and that's not in source control but it's pretty trivial. It should ideally be added to some Ansible config.